.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
```

Comment:
can we implement this job in the ci.yml workflow rather than having it in its own seen as it is part of the static code analysis step which is done in the ci.yml file

Reply:
sure